src/main/java/ru/bookcrossing/BookcrossingServer/BookCrossingServerApplication.java

```diff
@@ -18,6 +19,11 @@ public static void main(String[] args) {
 	public ModelMapper modelMapper() {
 		return new ModelMapper();
 	}
+
```

Comment:
It's better not to mix the main class and beans description. Place beans into separate configuration class